src/System.Windows.Forms/src/System/Windows/Forms/LinkLabel.Link.cs

```diff
@@ -42,6 +43,9 @@ internal Link(LinkLabel owner)
                 Owner = owner;
             }
 
+            internal LinkLabelLinkAccessibleObject AccessibleObject
+                => _accessibleObject ??= new(this, Owner);
+
```

Comment:
I think, we should pay attention that `Owner` can be null
```suggestion
            internal LinkLabelLinkAccessibleObject AccessibleObject
                => Owner is not null
                    ? _accessibleObject ??= new(this, Owner);
                    null;

```